flaky/_flaky_plugin.py

```diff
@@ -10,11 +10,16 @@
 class _FlakyPlugin(object):
     _retry_failure_message = ' failed ({0} runs remaining out of {1}).'
     _failure_message = ' failed; it passed {0} out of the required {1} times.'
+    _not_rerun_message = ' failed and was not selected for rerun.'
 
     def __init__(self):
         super(_FlakyPlugin, self).__init__()
         self._stream = StringIO()
 
+    @property
+    def stream(self):
```

Comment:
docstring? Yes... this is a "stream". But semantically what is this used for? 

Is it used publicly?